browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -58,15 +59,21 @@ using ntp_background_images::NTPCustomImagesSource;
 
 namespace {
 
-std::string GetSearchWidgetDefaultHost(PrefService* local_state) {
+std::string GetSearchWidgetDefaultHost(
+    PrefService* local_state,
+    regional_capabilities::RegionalCapabilitiesService* regional_capabilities) {
   constexpr char kBraveSearchHost[] = "search.brave.com";
   constexpr char kYahooSearchHost[] = "search.yahoo.co.jp";
   if (!local_state) {
     CHECK_IS_TEST();
     return kBraveSearchHost;
   }
```

Comment:
Maybe we can remove `local_state` param also and checking `regional_capabilities` here instead?

Reply:
Fixed!